subprojects/core/src/main/java/org/gradle/caching/internal/BuildCacheServiceProvider.java

```diff
@@ -53,6 +57,7 @@ public BuildCacheServiceProvider(BuildCacheConfigurationInternal buildCacheConfi
 
     public BuildCacheService createBuildCacheService() {
         if (!startParameter.isBuildCacheEnabled()) {
+            buildOperationExecutor.run(new FinalizeBuildCacheConfigurationBuildOperation(null, null));
```

Comment:
I don't think we should. The general pattern of build scans is that we show how the build “expressed” for that invocation. That is, not the entire potential model/configuration.